src/i18n.js

```diff
@@ -7,9 +7,16 @@ import LanguageDetector from 'i18next-browser-languagedetector'
 import en from 'i18next-icu/locale-data/en'
 import pt from 'i18next-icu/locale-data/pt'
 
+const localeData = [en, pt]
+
+export const localesList = [
+  { value: 'en', name: 'English' },
+  { value: 'pt', name: 'Portuguese' }
+]
```

Comment:
I think we want to remove this list and switch to `src/lib/languages.json` which can be generated by `npx -q @olizilla/lol public/locales > src/lib/languages.json

Reply:
You're 100% right!